src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -0,0 +1,131 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.AdditionalAnswers.returnsFirstArg;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderLineItemDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+
+@ExtendWith(MockitoExtension.class)
+class OrderServiceTest {
+
+	@Mock MenuDao menuDao;
```

Comment:
개인적으로 어노테이션은 상단에 위치하도록 컨벤션을 사용하고있습니다, 개인적인 의견이니 더 나아보이는 방법을 선택해주세요 :)
```suggestion
	@Mock 
	MenuDao menuDao;
```

Reply:
일관성있게 상단에 고정해놓겠습니다~!